jobs/wclouser-fxa-db-counts/wclouser_fxa_db_counts/main.py

```diff
@@ -0,0 +1,135 @@
+            UNION ALL
+        SELECT 'sent_emails' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_sent_emails_v1`
+            UNION ALL
+        SELECT 'session_tokens' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_session_tokens_v1`
+            UNION ALL
+        SELECT 'signin_codes' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_signin_codes_v1`
+            UNION ALL
+        SELECT 'totp' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_totp_v1`
+            UNION ALL
+        SELECT 'unblock_codes' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_unblock_codes_v1`
+            UNION ALL
+        SELECT 'unverified_tokens' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_unverified_tokens_v1`
+            UNION ALL
+        SELECT 'verification_reminders' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_verification_reminders_v1`
+            UNION ALL
+        (SELECT
+           "accounts_with_secondary_emails" AS table_name,
+            COUNT(DISTINCT accounts.uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+        JOIN
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON
+            accounts.uid = emails.uid
+        WHERE
+            emails.isPrimary = FALSE
+        )
+            UNION ALL
+        (SELECT
+            "accounts_with_unverified_emails" AS table_name,
+            COUNT(DISTINCT accounts.uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+        JOIN
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON
+            accounts.uid = emails.uid
+        WHERE
+            emails.isVerified = FALSE
+        )
+            UNION ALL
+        (
+        SELECT
+            "accounts_linked_to_google" AS table_name,
+            COUNT(uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_linked_accounts_v1`
+        WHERE
+            providerId=1 -- see LinkedAccountProviderIds at https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/types.ts
+        )
+            UNION ALL
+        (
+        SELECT
+            "accounts_linked_to_apple" AS table_name,
+            COUNT(uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_linked_accounts_v1`
+        WHERE
+            providerId=2 -- see LinkedAccountProviderIds at https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/types.ts
+        )
+    """
+    
+    client = bigquery.Client(project="moz-fx-fxa-prod")
```

Comment:
This probably won't work in Airflow because its service account can't use `moz-fx-fxa-prod`. Because we're only querying tables from `moz-fx-data-shared-prod`, it'd be best to use this project here. This could also be parameterized like in https://github.com/mozilla/docker-etl/blob/3e28c5149663c83ef25e5acfda6015d90242e3ce/jobs/influxdb-to-bigquery/influxdb_to_bigquery/main.py#L47